### ISSUE TITLE
Breton language not using mutation

### ISSUE THREAD

**Issue by jbleduigou** [iamkun/dayjs#1104]
**Describe the bug**
Breton language has some mutations.  
See [this page](https://en.wikipedia.org/wiki/Breton_mutations#:~:text=Like%20all%20modern%20Celtic%20languages,certain%20syntactic%20or%20morphological%20environments.) for more details.
This logic is in place in Moment.js but is currently missing in dayjs.

**Expected behavior**
See for instance the test result for this:
```
Summary of all failing tests
 FAIL  test/locale/br.test.js
  ● RelativeTime: Time from X

    expect(received).toBe(expected) // Object.is equality
    
    Expected value to be:
      "2 vloaz ʼzo"
    Received:
      "2 bloaz ʼzo"

      44 |     moment.locale('br')
      45 |     expect(dayjs().from(dayjs().add(t[0], t[1])))
    > 46 |       .toBe(moment().from(moment().add(t[0], t[1])))
      47 |     expect(dayjs().from(dayjs().add(t[0], t[1]), true))
      48 |       .toBe(moment().from(moment().add(t[0], t[1]), true))
      49 |   })
      
      at forEach (test/locale/br.test.js:46:8)
          at Array.forEach (<anonymous>)
      at Object.<anonymous> (test/locale/br.test.js:42:5)
```

**Information**
 - Day.js Version : dev branch
 - Running jest in console on MacOs

**Comment by jbleduigou** [comment]
See commented line in br.test.js, line 39

**Comment by iamkun** [comment]
I'm sorry that I did not get what's the mutation error is?

**Comment by jbleduigou** [comment]
Breton language has a paradigm similar to the [German Declension](https://en.wikipedia.org/wiki/German_declension).
What happens is that the first letter of a word might change depending on the previous word in the sentence.

For instance the word "year" is "bloaz" in breton.
"1 year" becomes "1 bloaz", no mutation here.
However "2 years" becomes "2 vloaz", 'b' is mutated to 'v'.

**Comment by jbleduigou** [comment]
Implemented within https://github.com/iamkun/dayjs/pull/1103

**Comment by iamkun** [comment]
fixed in #1103
